tools/topology/CMakeLists.txt

```diff
@@ -58,6 +58,7 @@ set(TPLGS
 	"sof-apl-dmic-a2ch-b4ch\;sof-apl-dmic-a2ch-b4ch"
 	"sof-apl-dmic\;sof-apl-dmic-4ch\;-DCHANNELS=4"
```

Comment:
@juimonen Please add ":" in commit message after topology. E.g: topology: add ...

Reply:
ok, added